docs/file-reference.md

```diff
@@ -268,103 +318,125 @@ Path based ingresses specify a path component that can be compared against a URL
 | www.example.com      | www.example.com/test | Yes (Matched by the host, not the ingress) |
 |                      | www.example.com      | Yes |
 
-### MountSpec
-```yml
-# <MountSpec>
-  volumeRef: db
-  mountPath: /app/store
-  volumeSubPath: foo/bar
-  readOnly: true
+### Mount
+
+```yaml
+...
+services:
+- name: foobar
+  ...
+  containers:
+    ...
+    mounts:
+    - volumeRef: db
+      mountPath: /app/store
+      volumeSubPath: foo/bar
+      readOnly: true
+    ...
+...
 ```
 
 #### volumeRef
-| type | required | possible values                                                                 |
+
+| Type | Required | possible values                                                                 |
 |------|----------|---------------------------------------------------------------------------------|
 |string|    yes   | should conform to the definition of a subdomain in DNS (RFC 1123), [details](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/identifiers.md). |
 
 Should match the name of volume from root level [`volumes`](#volumes) directive or `service` level [`emptyDirVolumes`](#emptydirvolumes) directive.
 
 #### mountPath
-| type | required |
+
+| Type | Required |
 |------|----------|
 |string|    yes   |
 
 Absolute path within the container at which the volume should be mounted. Must not contain ':'.
 
 #### volumeSubPath
-| type | required | Default value      |
+
+| Type | Required | Default value      |
 |------|----------|--------------------|
 |string|    no    | "" (volume's root) |
 
 Path within the volume from which the container's volume should be mounted. Defaults to "" (volume's root).
 
 #### readOnly
-| type | required | Default value |
+
+| Type | Required | Default value |
 |------|----------|---------------|
 |bool  |    no    | `false`       |
 
 Volume is mounted read-only if `true`, read-write otherwise.
 
 
-### EmptyDirVolumeSpec
-Describes one EmptyDir volume. This can be referenced from the [container mounts](#mountspec).
+### EmptyDirVolume
+
+Describes one EmptyDir volume. This can be referenced from the [container mounts](#mount).
 
-```yml
-# <EmptyDirVolumeSpec>
-  name: tmp
+```yaml
+services:
+- name: foobar
+  ...
+  emptyDirVolumes:
+  - name: temp
 ```
 
 #### name
-| type | required |
+
+| Type | Required |
 |------|----------|
 |string|    yes   |
 
 Name of the EmptyDir volume. 
 
 
 ## volumes
```

Comment:
This is top level section. It should have same headline as `services` and `version` and you prepend others with "Section: ".
It should be same "Section: Volumes"